_data/harnesses/count-messages/kafka.yml

```diff
@@ -0,0 +1,55 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/01a-init.sh
+          render:
+            file: tutorials/count-messages/kafka/markup/dev/01a-init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/count-messages/kafka/markup/dev/02a-make-docker-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/02b-docker-compose-up.sh
+          render:
+            file: tutorials/count-messages/kafka/markup/dev/02b-start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/02c-wait-for-containers.sh
+          render:
+            skip: true
```

Comment:
For your consideration - render the command for waiting as you did in the ksqlDB version. 
I needed to run the `kafkacat` command twice the first time I got this 
```
ERROR: Topic pageviews error: Broker: Unknown topic or partition
       0
```
I realize it's a very minor point.

Reply:
Good catch. I've added the instruction in.